src/Tribe/Integrations/WPML/Category_Translation.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+
+/**
+ * Class Tribe__Events__Integrations__WPML__Category_Translation
+ *
+ * Translates category links on the site front-end.
+ */
+class Tribe__Events__Integrations__WPML__Category_Translation {
+
+	/**
+	 * @var static
+	 */
+	protected static $instance;
+
+	/**
+	 * @return Tribe__Events__Integrations__WPML__Category_Translation
+	 */
+	public static function instance() {
+		if ( empty( self::$instance ) ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Filters the `tribe_events_category_slug` to return the category slug that's WPML aware.
+	 *
+	 * WPML does not currently support translation of custom taxonomies root ,e.g. `category` in
+	 * The Events Calendar case. But we do take WPML-managed translations of the `category` slug
+	 * into account in our rewrite rules and try to show a localized version of the `category` slug
+	 * in the permalinks.
+	 *
+	 * @param string $slug The original, possibily translated, category slug.
+	 *
+	 * @return string The category slug in its ENG form if the Events Category translation is not active
+	 *                or in a translation that The Events Calendar supports.
+	 */
+	public function filter_tribe_events_category_slug( $slug ) {
+		/** @var SitePress $sitepress */
+		global $sitepress;
+
+		$tax_sync_options     = $sitepress->get_setting( 'taxonomies_sync_option' );
+		$translate_event_cat  = ! empty( $tax_sync_options[ Tribe__Events__Main::TAXONOMY ] );
+		$using_lang_query_var = $sitepress->get_setting( 'language_negotiation_type' ) == 3;
+
+		if ( !$translate_event_cat || $using_lang_query_var ) {
+			$slug = 'category';
+		} else {
+			$lang        = $sitepress->get_locale( ICL_LANGUAGE_CODE );
```

Comment:
We could either remove the extra whitespace before `=` or else align the following lines.